benchmarks/src/testSetup/testUtils.ts

```diff
@@ -19,8 +20,8 @@ import { Platform } from 'react-native';
 
 export const DEFAULT_ENV_TEST_CONFIG: TestConfig = {
     command: Command.Start,
-    scenario: Config.BENCH_SCENARIO,
-    runType: Config.BENCH_RUN_TYPE,
+    scenario: Config.BENCH_SCENARIO || undefined,
+    runType: Config.BENCH_RUN_TYPE || RunType.BASELINE,
```

Comment:
Non-baseline runs gets instrumented so we default to baseline in case this is undefined just to be sure.